### DISCUSSION TITLE
Adding the attackers/victims option to switch between in Grip Alarms

### DISCUSSION THREAD

**Discussion by moawnallah** [InternetHealthReport/gsoc#24] [General]
@romain-fontugne This is related [Adding switches between originasn/asn for AS hegemony alarms and start/endpoint for network delay alarms](https://github.com/orgs/InternetHealthReport/projects/10/views/1?pane=issue&itemId=37684920).

**Comment by romain-fontugne** [comment]
yes, I think that would be helpful for GRIP alarms too
